app/frontend/src/components/Buttons/Button.tsx

```diff
@@ -47,6 +47,7 @@ function Button({
       )}
       href={props.href}
       target={props.href ? props.target : undefined}
+      rel={props.href ? "noopener noreferrer" : undefined}
```

Comment:
@Aveline-art could you help me understand why we want to check if there is an href first, instead of just setting rel to "noopener noreferrer" all the time?

Reply:
The check is needed because the `rel` attribute only exists for `<a>` and not `<button>` tags. I think, technically, adding it to buttons would not cause any issues in rendering, however.